chart/README.md

```diff
@@ -175,6 +175,7 @@ Specify each parameter using the `--set key=value[,key=value]` argument to `helm
 | `auditLog.maxSize`                       | 100                                                                       | ***int*** - maximum size in megabytes of the audit log file before it gets rotated (only applies when **auditLog.destination** is set to **hostPath**)                                                                                                                                  |
```

Comment:
> "Images must use digests instead of tags. This ensures CNAB building is deterministic."

I don't see this on the page anymore: https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-technical-assets-kubernetes

I see this though: 
> When referencing images, you can utilize tags or digests. 

Did it change ? 

I was trying to check the scope of this change, would all rancher images need to use digests or just ones specified in the chart for install ?

Reply:
Yes, looks like their are no longer insisting on image digest only, per https://learn.microsoft.com/en-us/partner-center/marketplace/azure-container-technical-assets-kubernetes?tabs=linux%2Clinux2#update-the-helm-chart

```When referencing images, you can utilize tags or digests. However, it's important to note that the images will internally be re-tagged to point to the Microsoft-owned Azure Container Registry (ACR). When you update a tag, a new version of the CNAB must be submitted to the Azure Marketplace. This is so that the changes can be reflected in customer deployments.
```
We'll need to test it out.